CONTRIBUTING.md

```diff
@@ -11,19 +11,18 @@ Simple Icons welcomes contributions and corrections. Before contributing, please
 
 ## Requesting An Icon
 
-We welcome icon requests. However, before you submit a new issue please make sure the icon:
+We welcome icon requests, before you submit a new issue please make sure the icon:
 
 * Is of a _popular_ brand.
 * Has not already been requested.
-* Can be represented in a monochromatic format.
 
-When submitting a request for a new or updated icon, please submit the issue with the following information:
+When submitting a request for a new or updated icon include helpful information such as:
```

Comment:
`When submitting a request for a new or updated icon, please include helpful information such as:`
Sounds a little more casual (and less demanding) to me. :)

Reply:
It is interesting how we have a different view on this 😅 I personally feel like including "please" is kinda demeaning. I think we should get a third opinion (@birjolaxew) on this and just go with that

Reply:
I guess [that](https://github.com/simple-icons/simple-icons/pull/644#pullrequestreview-70020565) settles it 🙂